doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -257,17 +257,31 @@ Connectivity Bridge
 nRF5340 Audio
 -------------
 
-* Added CAP initiator for the Bluetooth LE Audio unicast (CIS) client.
+* Added:
+
+  * CAP initiator for the Bluetooth LE Audio unicast (CIS) client.
+  * Support for any context type, not just media.
+  * Rejection of connection if unicast_client/broadcast_source tries to use an unsupported sample rate.
+  * Missing data to advertising packet after directed advertising has timed out.
+  * Debug prints of discovered endpoints.
+  * Support for multiple unicast_servers in unicast_client, regardless of location.
+  * Common Audio Profile (CAP) to unicast_client, including Coordinated Set Identification Profile (CSIP).
```

Comment:
Is this a partial repeat of the first point?

Reply:
Fixed